### DISCUSSION TITLE
expand layout issue CRUD::enableDetailsRow()

### DISCUSSION THREAD

**Discussion by NeguinhoAdmin** [Laravel-Backpack/community-forum#1054] [General]
Can someone please tell me how to fix that overlap issues.

<img width="216" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/136070103/e25422d6-2349-4df8-a002-7bb0511baee2">

**Comment by karandatwani92** [comment]
Hey @NeguinhoAdmin 

Thanks for pointing out the issue. It's a bug in coreui-v4. I've escalated the issue. https://github.com/Laravel-Backpack/community-forum/issues/1055

**Reply by NeguinhoAdmin** [reply]
Thank you, let me know when it is done. 

Secondly, how to get any updates, non-breaking version

**Reply by karandatwani92** [reply]
Yes, we'll inform you; you can also subscribe to the issue to get updates.

What do you mean by updates?
- for version updates, `composer update` will help.
- To know what keeps adding to the Backpack. Subscribe to the Backpack [newsletter](https://backpackforlaravel.com/newsletter) and get monthly progress reports, tutorials, tips, and more.
